### DISCUSSION TITLE
Two years later and the community page still isn't up-to-date

### DISCUSSION THREAD

**Discussion by thatkookooguy** [vmware/clarity#5162] [General] [answered]
Regarding the previous issue I opened about this **two years ago** (issue #2192), I see that still the community page wasn't updated (based on the fact that I don't see my name. if there are any other explanations, let me know).

Is this planned at all? I don't want to nag. But last time the discussion finished with ppl saying it's planned for the near future

@gnomeontherun @adityarb88

**Comment by gnomeontherun** [comment]
Hello @Thatkookooguy we're in the process of replacing our website and will not longer have this page. We will rely on the GitHub community metrics to show who collaborates and contributes, because keeping this page up to date was hard to do. You can see the list here https://github.com/vmware/clarity/graphs/contributors

**Comment by thatkookooguy** [comment, marked as answer]
You can have an automatic job that extracts the contributors automatically and make this pain free.
For example, [`all-contributors` is used as a GitHub action](https://github.com/marketplace/actions/all-contributors-action) in order to add people to the contributors' section of a readme.
[You can also add it to an HTML page using `all-contributors`](https://allcontributors.org/docs/en/specification).
The problem of updating contributors by hand is already solved :-)

But I guess removing it is fine as well...
